.dockstore.yml

```diff
@@ -87,6 +87,27 @@ workflows:
      filters:
        branches:
          - master
+   - name: ngs_filter_extract
+     subclass: WDL
+     primaryDescriptorPath: /scripts/variantstore/wdl/ngs_filter_extract.wdl
+     testParameterFiles:
+       - /scripts/variantstore/wdl/ngs_filter_extract.inputs.json
+     filters:
+       branches:
+         - master
+         - ah_var_store
+         - kc_feature_tieout
```

Comment:
we can remove any references to branches other than master and ah_var_store (here and for cohort extract)